profile/src/test/java/com/scottlogic/deg/profile/reader/JsonProfileReaderTests.java

```diff
@@ -585,21 +583,22 @@ public void shouldAllowValidISO8601DateTime() throws IOException {
 
         expectRules(
             ruleWithConstraints(
+
                 typedConstraint(
-                    IsAfterOrEqualToConstantDateTimeConstraint.class,
+                    IsBeforeConstantDateTimeConstraint.class,
```

Comment:
Why was this swapped around?

Reply:
the test cares about the order, but the order isn't guaranteed. this will have to be changed occasionally when json profile reader changes